src/main/resources/tags/info/mods.tag

```diff
@@ -0,0 +1,5 @@
+type: text
+
+---
+
+Geyser emulates a Vanilla Java game client. If the mods installed on the server allow a full vanilla client to connect, then a Geyser player can. (Conversely, if a player must install mods, a Geyser player can not join.)
```

Comment:
Nitpick: `Vanilla Java game client` -> `vanilla Java game client`.

Reply:
I personally have seen it "branded" like that a few times to distinguish it from non-vanilla server software.

Reply:
Vanilla isn't a proper term, it's just a general term to clarify a non-modified client. It's more of an adjective to "Java" than "Vanilla Java" is one to "game client."